_includes/header.html

```diff
@@ -0,0 +1,14 @@
+<header class="site-header">
+    <a class="site-title" href="{{ site.baseurl }}/">
+        {{ site.title }}
+    </a>
+    <nav class="site-nav">
+        {% for page in site.pages %}
+            {% if page.title %}
```

Comment:
You might be able to use an `index.md` instead of html by using a check here for it is `index` or something.